freud/box.pyx

```diff
@@ -291,6 +291,7 @@ cdef class Box:
         Args:
             vecs (:math:`\left(3, \right)` or :math:`\left(N, 3\right)` :class:`numpy.ndarray`):
                 Unwrapped vector(s).
+            inplace
```

Comment:
You'll need to write a docstring describing this argument. If you're unsure how to document something that isn't a numpy array, have a look at the `is2D` argument to the box constructor.